app/main/tilt.py

```diff
@@ -68,23 +68,33 @@ def tilts(devices):
     return tilts
 
 
-async def scan(app):
-    # print("Scanning for tilts...")
+async def scan(app, interval):
     with app.app_context():
+        # app.logger.debug("Scanning for tilts...")
         devices = await BleakScanner.discover(10.0)
-        for tilt in tilts(devices):
-            # print(tilt)
-            process_tilt_data(tilt)
+        try:
+            tilts_found = tilts(devices)
+            for tilt in tilts_found:
+                # app.logger.debug(tilt)
+                process_tilt_data(tilt)
 
+            app.logger.debug(f"found {len(tilts_found)} tilt(s)")
```

Comment:
Not sure if you will consider this excessive, but it will log about 6 times per minute when tilts are in use, which was why i kept it commented out for general use.

Reply:
debug is default disabled so results with nothing unless you do this:

```FLASK_ENV=development python3 ...` to start the server

Reply:
Thanks, I didn't realize this (or remember) maybe thats why i was using print. Doh!